questions/services.py

```diff
@@ -718,13 +724,13 @@ def sorting_key(q: Question):
                 case "numeric" | "date":
                     return aggregation.centers[0]
                 case "multiple_choice":
-                    return aggregation.forecast_values[0]
+                    return max(aggregation.forecast_values)
```

Comment:
group MC questions don't exist, but if they do, filtering by the subquestions that have the highest confidence in a single outcome seems appropriate.